bitnami/suitecrm/README.md

```diff
@@ -50,74 +50,128 @@ The command removes all the Kubernetes components associated with the chart and
+| `suitecrmLastName`                          | Last name                                                                                             | `Last`                                                       |
+| `suitecrmSmtpHost`                          | SMTP host                                                                                             | `nil`                                                        |
+| `suitecrmSmtpPort`                          | SMTP port                                                                                             | `nil`                                                        |
+| `suitecrmSmtpUser`                          | SMTP user                                                                                             | `nil`                                                        |
+| `suitecrmSmtpPassword`                      | SMTP password                                                                                         | `nil`                                                        |
+| `suitecrmSmtpProtocol`                      | SMTP protocol [`ssl`, `tls`]                                                                          | `nil`                                                        |
+| `suitecrmValidateUserIP`                    | Whether to validate the user IP address or not                                                        | `no`                                                         |
+| `allowEmptyPassword`                        | Allow DB blank passwords                                                                              | `yes`                                                        |
+| `persistence.enabled`                       | Enable persistence using PVC                                                                          | `true`                                                       |
+| `persistence.storageClass`                  | PVC Storage Class for SuiteCRM volume                                                                 | `nil` (uses alpha storage class annotation)                                                  |
+| `persistence.existingClaim`                 | An Existing PVC name for SuiteCRM volume                                                              | `nil` (uses alpha storage class annotation)                                                  |
+| `persistence.accessMode`                    | PVC Access Mode for SuiteCRM volume                                                                   | `ReadWriteOnce`                                              |
+| `persistence.size`                          | PVC Storage Request for SuiteCRM volume                                                               | `8Gi`                                                        |
```

Comment:
Create a "Persistence parameters" section for these parameters.

Reply:
You can include the "volumePermissions" parameters (that you are missing) in this new section too.